src/components/DataView/DataView.tsx

```diff
@@ -389,33 +409,35 @@ function DataView (props: DataViewProps): ReactElement  {
 					onActiveFiltersChange={props.onActiveFiltersChange}
 				/>
 			</div>
-			<div className="headerRow filters">
+			<div className={display === "list" ? "headerList" : "headerRow filters"}>
+				<div className="left">

```

Comment:
I'm not sure why the DataViewFilters were moved here... if you load the storybook you'll see they appear twice 
![image](https://user-images.githubusercontent.com/23409284/202560320-847d069c-339c-4951-8aad-fea27d6c0c88.png)

They should only appear in the DataViewTitleBar which is part of the other ticket you were working on.

Also I'm not sure what the .headerList class is needed for.